sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -158,6 +164,14 @@ def test_regressor_parameter_checks():
                          " or an integer. 'invalid' was passed",
                          GradientBoostingRegressor(n_iter_no_change='invalid')
                          .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
+                         GradientBoostingRegressor(loss='huber',
+                                                   presort='invalid')
+                         .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
```

Comment:
This one seems also unnecessary? Also, why do you need to specify loss? It seems only a test for presort?